CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## V 0.7.0
+- Update Error classification mechanism to use error message as first option
```

Comment:
Is this a fix or a breaking change?

Reply:
it doesn't affect the current behavior, updating to 0.6.2